### PR TITLE
Allow setting tag when posting review

### PR DESCRIPTION
**Pull request by syntonyze**
By calling the set_tag method it is now possible to set a specific
tag when posting a review.

### CODE REVIEW

File: pygerrit2/rest/__init__.py [hunk 1]
```diff
@@ -307,6 +308,14 @@ def set_message(self, message):
         """
         self.message = message
 
+    def set_tag(self, tag):
```
Comment (dpursehouse) [inline pygerrit2/rest/__init__.py]: github won't let me add a comment down there, but please also add handling of `self.tag` in the `__str__` method (L369).

File: pygerrit2/rest/__init__.py [hunk 2]
```diff
@@ -282,9 +282,10 @@ class GerritReview(object):
 
     """
 
-    def __init__(self, message=None, labels=None, comments=None):
+    def __init__(self, message=None, labels=None, comments=None, tag=None):
```
Comment (dpursehouse) [inline pygerrit2/rest/__init__.py]: Please also add a docstring for `tag`

**Comment by syntonyze** [comment]
> Aside from the inline comments, can you confirm that this commit was intentionally done with the corporate email address?

Not very important, but I actually did not realize that. I have now amended with my personal one.

File: pygerrit2/rest/__init__.py (hunk 2 above)
Comment (syntonyze) [reply pygerrit2/rest/__init__.py]: Done

File: pygerrit2/rest/__init__.py (hunk 1 above)
Comment (syntonyze) [reply pygerrit2/rest/__init__.py]: Done

**Comment by dpursehouse** [comment]
Thanks!

**Pull request closed without merging**